uint8 ID
float32 CURRENT
float32 RPM